include/mbedtls/ctr_drbg.h

```diff
@@ -15,6 +15,8 @@
  *   (`MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` not set) and
  *   with #MBEDTLS_CTR_DRBG_ENTROPY_LEN set to 48 or more.
  * - 256 bits if AES-256 is used, #MBEDTLS_CTR_DRBG_ENTROPY_LEN is set
+ *   to 32 or more, and the DRBG is initialized with an explicit
+ *   nonce in the \c custom parameter to see mbedtls_ctr_drbg_seed().
```

Comment:
Maybe it is just my English, but I have trouble interpreting this sentence. I would find it easier to read like this:
```diff
- *   nonce in the \c custom parameter to see mbedtls_ctr_drbg_seed().
+ *   nonce in the \c custom parameter (see mbedtls_ctr_drbg_seed()).
```

Reply:
I meant to write “the `custom` parameter to `mbedtls_ctr_drbg_seed()`”. Fixed. Or should this be “the `custom` parameter passed to `mbedtls_ctr_drbg_seed()`”?